packages/exec/__tests__/exec.test.ts

```diff
@@ -287,8 +287,6 @@ describe('@actions/exec', () => {
   })
 
   it('Handles child process holding streams open', async function() {
-    // this was timing out on some slower hosted macOS runs at default 5s
-    jest.setTimeout(10000)
```

Comment:
From my testing it didn't appear these actually did anything (I think you need to set them outside the test). I moved them to the end of the method as the third arg, which works.